content/pt/docs/concepts/configuration/pod-overhead.md

```diff
@@ -0,0 +1,197 @@
+
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+No Kubernetes, a sobrecarga de _Pods_ é definido no tempo de
+[admissão](/docs/reference/access-authn-authz/extensible-admission-controllers/#what-are-admission-webhooks)
+de acordo com a sobrecarga associada à
+[RuntimeClass](/docs/concepts/containers/runtime-class/) do _Pod_.
+
+Quando é ativada a Sobrecarga de Pod, a sobrecarga é considerada adicionalmente à soma das
+solicitações de recursos do _container_ ao agendar um Pod. Semelhantemente, o _kubelet_
+incluirá a sobrecarga do Pod ao dimensionar o cgroup do Pod e ao
+executar a classificação de despejo do Pod.
+
+## Possibilitando a Sobrecarga do Pod {#set-up}
+
+Terá de garantir que o [portão de funcionalidade](/docs/reference/command-line-tools-reference/feature-gates/)
+`PodOverhead` está ativo (está ativo por defeito a partir da versão 1.18)
+por todo o cluster, e uma `RuntimeClass` é utilizada que defina o campo `overhead`.
+
+## Exemplo de uso
+
+Para usar a funcionalidade PodOverhead, é necessário uma RuntimeClass que define o campo `overhead`.
+Por exemplo, poderia usar a definição da RuntimeClass abaixo com um _container runtime_ virtualizado
+que usa cerca de 120MiB por Pod para a máquina virtual e o sistema operativo convidado:
+
+```yaml
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1beta1
+metadata:
+    name: kata-fc
+handler: kata-fc
+overhead:
+    podFixed:
+        memory: "120Mi"
+        cpu: "250m"
+```
+
+As cargas de trabalho que são criadas que especifiquem o manipulador RuntimeClass `kata-fc` irão
+ter a sobrecarga de memoria e cpu em conta para os cálculos da quota de recursos, agendamento de nós,
+assim como dimensionamento do cgroup do Pod.
+
+Considere executar a seguinte carga de trabalho de exemplo, test-pod:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: test-pod
+spec:
+  runtimeClassName: kata-fc
+  containers:
+  - name: busybox-ctr
+    image: busybox
+    stdin: true
+    tty: true
+    resources:
+      limits:
+        cpu: 500m
+        memory: 100Mi
+  - name: nginx-ctr
+    image: nginx
+    resources:
+      limits:
+        cpu: 1500m
+        memory: 100Mi
+```
+
+Na altura de admissão o [controlador de admissão](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/) RuntimeClass
+atualiza o _PodSpec_ da carga de trabalho de forma a incluir o `overhead` como descrito na RuntimeClass. Se o _PodSpec_ já tiver este campo definido
+o _Pod_ irá ser rejeitado. No exemplo dado, como apenas o nome RuntimeClass é especificado, o controlador de admissão muda o _Pod_ de forma a
+incluir um `overhead`.
+
+Depois do controlador de admissão RuntimeClass, pode verificar o _PodSpec_ atualizado:
+
+```bash
+kubectl get pod test-pod -o jsonpath='{.spec.overhead}'
+```
+
+O output é:
+```
+map[cpu:250m memory:120Mi]
+```
+
+Se for definido um _ResourceQuota_, a soma dos pedidos aos _containers_ assim como o campo `overhead` são contados.
```

Comment:
```suggestion
Se for definido um _ResourceQuota_, a soma dos pedidos dos _containers_ assim como o campo `overhead` são contados.
```

Reply:
I think here it means "pedidos aos containers" as in the requests made to the containers. What do you think?

Reply:
I think that the requests are part of the container. Like in pt: "os requisitos dos conteiners". 

I understand that the resources are part of the definition of the container and theses resources definitions are sent to the Node.
The way you wrote, for me, looks like that, the request is sent to the container.

I propose another translation: "Se um ResourceQuota for definido, são contados a soma dos pedidos dos containers bem como do campo overhead."